package.json

```diff
@@ -17,13 +14,16 @@
     "test:watch": "jest --watch",
     "test:cover": "jest --coverage",
     "lint": "eslint \"src/**/*.js\"",
-    "start": "parcel ./src/index.html"
+    "start": "react-scripts start",
```

Comment:
you are missing a lot of config.. like babel, webpack and hot reload to make this easy